src/360-core/browzine-360-core-adapter.js

```diff
@@ -55,15 +55,15 @@ browzine.serSol360Core = (function() {
     return (title && title.title) ? title.title : '';
   };
 
-  function getTarget(title) {
-    var issn = getIssn(title).toLowerCase().trim();
-
-    var element = $(".results-identifier").filter(function() {
-      return $.trim($(this).text()).toLowerCase().indexOf(issn) > -1 && issn.length > 0;
-    }).closest(".results-title-row");
-
-    var target = element.length > 0 ? element[0] : undefined;
+  function getTarget(index) {
+    console.log(index, 'our index inside getTarget');
+    var elements = $(".results-identifier").closest(".results-title-row");
 
+    if (index >= elements.length) {
+      var target = undefined;
+    } else {
+      var target = elements[index];
+    }
```

Comment:
I think this should work - if you wanted to keep the ternary approach, you could also just do:

```
var target = element.length > index ? element[index] : undefined;
```

Since index is 0-based, and length is 1 based, then it should work fine.